src/main/java/me/xmrvizzy/skyblocker/skyblock/item/PriceInfoTooltip.java

```diff
@@ -20,105 +22,130 @@
+                            .append(getCoinsMessage(getItem.get("sellPrice").getAsDouble(), count)));
+                }
             }
 
-            if (SkyblockerConfig.get().general.itemTooltip.enableMuseumDate
-                    && !listString.contains("Museum") && isMuseumJson != null && isMuseumJson.has(name)) {
-
-                String itemCategory = isMuseumJson.get(name).toString().replaceAll("\"", "");
-                String format = switch (itemCategory) {
-                    case "Weapons" -> "%-18s";
-                    case "Armor" -> "%-19s";
-                    default -> "%-20s";
-                };
-
-                lines.add(new LiteralText(String.format(format, "Museum: (" + itemCategory + ")"))
-                        .formatted(Formatting.LIGHT_PURPLE)
-                        .append(new LiteralText(timestamp != null ? timestamp : "").formatted(Formatting.RED)));
-            } else if (!listString.contains("Obtained") && timestamp != null
-                    && SkyblockerConfig.get().general.itemTooltip.enableMuseumDate) {
-
+            if (SkyblockerConfig.get().general.itemTooltip.enableMuseumDate && !listString.contains("Museum")) {
+                if (isMuseumJson.has(name)) {
+                    String itemCategory = isMuseumJson.get(name).toString().replaceAll("\"", "");
+                    String format = switch (itemCategory) {
+                        case "Weapons" -> "%-18s";
+                        case "Armor" -> "%-19s";
+                        default -> "%-20s";
+                    };
+
+                    lines.add(new LiteralText(String.format(format, "Museum: (" + itemCategory + ")"))
+                            .formatted(Formatting.LIGHT_PURPLE)
+                            .append(new LiteralText(timestamp != null ? timestamp : "").formatted(Formatting.RED)));
+                }
+            } else if (!listString.contains("Obtained") && timestamp != null && SkyblockerConfig.get().general.itemTooltip.enableMuseumDate) {
                 lines.add(new LiteralText(String.format("%-22s", "Obtained: "))
                         .formatted(Formatting.LIGHT_PURPLE)
                         .append(new LiteralText(timestamp).formatted(Formatting.RED)));
             }
-        } catch (Exception e) {
+        } catch (NullPointerException ex) {
+            if ((npcPricesJson == null || bazaarPricesJson == null || oneDayAvgPricesJson == null
+                    || threeDayAvgPricesJson == null || lowestPricesJson == null || isMuseumJson == null)
+                    && !nullMsgSend)  {
+                MinecraftClient.getInstance().player.sendMessage(new TranslatableText("skyblocker.itemTooltip.NullMessage"), false);
+                nullMsgSend = true;
+                skyblocker.scheduler.schedule(() -> nullMsgSend = false, 1200);
+            }
+        } catch (Exception ex) {
             if (MinecraftClient.getInstance().player == null) {
```

Comment:
This if block will only be entered if null pointer exception wasn't the exception that was caught, due to block above catching all of them (null pointer exceptions).
We will definitely want to see the reason for the original exception if it ever occurs.

Reply:
I forgot to add printStackTrace. And it will be better to add "Check latest.log for detailed information." to nullMessage.